components/NavBar/DropdownSignOut/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Fragment } from 'react'
+import { Menu, Transition } from '@headlessui/react'
+import { useFirebase } from '@components/FirebaseContext'
+
+const DropdownSignOut = () => {
+  const { user, signOutClearDataUnlockGrid } = useFirebase()
+  return (
+    <Menu as='div' className='relative flex text-xl text-right'>
+      <Menu.Button className='px-3 ml-8 font-serif text-white rounded hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus:ring-opacity-50'>
+        <span className='capitalize'>{user?.displayName}</span>
+        <svg
+          xmlns='http://www.w3.org/2000/svg'
+          className='inline w-5 h-5 ml-2 -mr-1'
+          aria-hidden='true'
+          viewBox='0 0 20 20'
+          fill='currentColor'
+        >
+          <path
+            fillRule='evenodd'
+            d='M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z'
+            clipRule='evenodd'
+          />
+        </svg>
+      </Menu.Button>
+      <Transition>
+        {/* Arrow */}
+        <Transition.Child
+          as={Fragment}
+          enter='transition ease-in-out duration-150 transform'
+          enterFrom='translate-y-full opacity-0'
+          enterTo='translate-y-0 opacity-100'
+          leave='transition ease-in-out duration-75 transform'
+          leaveFrom='translate-y-0 opacity-100'
+          leaveTo='translate-y-full opacity-0'
+        >
+          <div className='absolute right-[0.62rem] top-8'>
+            <div className='w-0 h-0 border-b-orange border-b-[1em] border-r-transparent border-r-[0.5rem] border-l-transparent border-l-[0.5rem]' />
+          </div>
+        </Transition.Child>
+        {/* Menu Items */}
+        <Transition.Child
+          as={Fragment}
+          enter='transition-opacity ease-linear duration-75'
+          enterFrom='opacity-0'
+          enterTo='opacity-100'
+          leave='transition-opacity ease-linear duration-100'
+          leaveFrom='opacity-100'
+          leaveTo='opacity-0'
+        >
+          <Menu.Items className='absolute right-[0.5rem] w-28 font-serif top-[3rem] focus:outline-none'>
```

Comment:
Fairly sure there's tailwind classes already available instead of the JIT ones declared here